### PR TITLE
ports knife boxing gloves and knife boxing martial arts from hippie (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
https://github.com/HippieStation/HippieStation/pull/7690 original pr
"can be constructed out of 1 pair of boxing gloves of any color (they all become red due to the influence of Armok on this plane of reality) and 2 knives. does between 9-13 brute damage (for humans)" (from original pr)

#### Changelog
// maintainer edit: fixed changelog
:cl:  zamolxius
rscadd: ported knife boxing gloves and its corresponding martial arts  
/:cl:

### CODE REVIEW

File: code/datums/martial/knifeboxing.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/datum/martial_art/knifeboxing/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	to_chat(A, "<span class='warning'>Can't grab while knife-boxing!</span>")
+	return TRUE
+
+/datum/martial_art/knifeboxing/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+
+	var/atk_verb = pick("left hook","right hook","straight punch")
+
```
Comment (JamieD1) [inline code/datums/martial/knifeboxing.dm:17]: Dont get why there are random spaces here

**Review by JamieD1 — APPROVED** [review]

File: code/datums/martial/knifeboxing.dm (hunk 1 above)
Comment (boodaliboo) [reply code/datums/martial/knifeboxing.dm:17]: i dunno i just ported code and it compiles

**Comment by alexkar598** [comment]
Enables greytiding

**Comment by Hopekz** [comment]
reopening for traitor recipe

**Review by Hopekz — APPROVED** [review]
Resolved your merge conflict for you. All good.

**Pull request merged**